src/builder/QuarkBuilder.sol

```diff
@@ -326,41 +325,43 @@ contract QuarkBuilder {
             }
 
             assertSufficientPaymentTokenBalances(
-                actions, chainAccountsList, borrowIntent.chainId, supplementalPaymentTokenBalance
+                actionsArray, chainAccountsList, borrowIntent.chainId, supplementalPaymentTokenBalance
             );
         }
 
         // Merge operations that are from the same chain into one Multicall operation
-        (quarkOperations, actions) = QuarkOperationHelper.mergeSameChainOperations(quarkOperations, actions);
+        (quarkOperationsArray, actionsArray) =
+            QuarkOperationHelper.mergeSameChainOperations(quarkOperationsArray, actionsArray);
 
         // Wrap operations around Paycall/Quotecall if payment is with token
         if (payment.isToken) {
-            quarkOperations =
-                QuarkOperationHelper.wrapOperationsWithTokenPayment(quarkOperations, actions, payment, useQuotecall);
+            quarkOperationsArray = QuarkOperationHelper.wrapOperationsWithTokenPayment(
+                quarkOperationsArray, actionsArray, payment, useQuotecall
+            );
         }
 
         // Construct EIP712 digests
         EIP712Helper.EIP712Data memory eip712Data;
-        if (quarkOperations.length == 1) {
+        if (quarkOperationsArray.length == 1) {
```

Comment:
same comment as above; lines 343-359 no longer exist on main. You should be able to rebase and then remove those changes

Reply:
Oh I see.  Thanks! I have rebased it so this change is now gone 👍